### ISSUE TITLE
代码里展示的是 Softmax Regression，而非 Logistic Regression

### ISSUE THREAD

**Issue by wangyixu** [yunjey/pytorch-tutorial#192]
pytorch-tutorial/tutorials/01-basics/logistic_regression/main.py

`# Logistic regression model
model = nn.Linear(input_size, num_classes)

# Loss and optimizer
# nn.CrossEntropyLoss() computes softmax internally
criterion = nn.CrossEntropyLoss()  
`

**Comment by OrangeC93** [comment]
难道不就是用 softmax regression 嘛 (which is used for multi-class classification problems)

**Comment by yuzi1949** [comment]
是的，没太看明白，注释里写的是Logistic Regression

**Comment by nuaa030710312** [comment]
看来不是我一个人有疑问
